static/js/settings_org.js

```diff
@@ -780,14 +783,17 @@ export function build_page() {
 
                         break;
                     }
+                    case "any_time": {
+                        data.allow_message_deleting = true;
+                        data.message_content_delete_limit_seconds = JSON.stringify("anytime");
+
```

Comment:
we can also change the value of dropdown from "any_time" to "anytime", but edit limit setting also uses same object of `settings_config`, and we would also want to do the same refactoring of making zero invalid for that setting too, so this change can be done with that only.

Reply:
OK, makes sense.  I'd consider noting this TODO in a comment.

Reply:
So, I removed the TODO comment for now, since I wanted to ask whether we should instead rename it to `unlimited`?